esmvalcore/preprocessor/_derive/sispeed.py

```diff
@@ -0,0 +1,55 @@
+"""Derivation of variable `sispeed`."""
+
+from iris import Constraint
+from iris.coords import DimCoord
+
+from ._baseclass import DerivedVariableBase
+
+
+class DerivedVariable(DerivedVariableBase):
+    """Derivation of variable `sispeed`."""
+
+    # Required variables
+    required = [
+        {'short_name': 'usi', },
+        {'short_name': 'vsi', }
+    ]
+
+    @staticmethod
+    def calculate(cubes):
+        """
+        Compute sispeed module from velocity components siu and siv.
+
+        Arguments
+        ----
+            cubes: cubelist containing velocity components.
+
+        Returns
+        -------
+            Cube containing sea ice speed.
+
+        """
+        siu = cubes.extract_strict(Constraint(name='sea_ice_x_velocity'))
+
+        siv = cubes.extract_strict(Constraint(name='sea_ice_y_velocity'))
+        siv.remove_coord('latitude')
+        siv.remove_coord('longitude')
+
+        # Models usually store siu and siv in slightly different points
```

Comment:
Maybe it would be nice to check if they are actually close, to avoid errors?

Reply:
Yes, will fix it

Reply:
Ok. I have been thinking in this:

- I will try to do it without doing anything with the coordinates.

- If not, I will do as it is now after a warning, unless the `np.allcose ` for more than 5 degrees fail. Why so big? The real expected values depend on the mesh and the resolution, so it is not easy to get a small estimate that will work well for all cases. I hope the big value will help us if we have other issues with the grid, although more of those will be caught by the checker